src/WebJobs.Extensions.Http/Routing/IWebJobsRouter.cs

```diff
@@ -11,6 +11,8 @@ public interface IWebJobsRouter : IRouter
 
         void AddFunctionRoutes(IRouter functionRoutes, IRouter proxyRoutes);
 
+        void AddCustomRoutes(IRouter customRoutes);
```

Comment:
@fabiocav This is a breaking changes to a public interface. Is there a way for us to make this change w/o breaking this interface?